recipeers/ingredientes.py

```diff
@@ -1,5 +1,15 @@
-class Ingredientes:
-  def __init__(self, nombre: set[str], cantidad: float,  medida: str):
-    self.nombre = nombre
-    self.cantidad = cantidad
-    self.medida = medida
\ No newline at end of file
+class Ingrediente:
+    def __init__(self, nombre: str):
+        self.nombre = nombre
+
+    def __repr__(self):
+        return f"Ingrediente({self.nombre})"
+    
+class CantidadIngrediente:
+    def __init__(self, ingrediente: Ingrediente, cantidad: float, unidad_medida: str):
+        self.ingrediente = ingrediente
+        self.cantidad = cantidad
+        self.unidad_medida = unidad_medida
```

Comment:
¿Diferentes cantidades van a tener diferentes medidas? ¿O la medida va a depender sólo del ingrediente?

Y en todo caso que quisiérais hacerlo así, ¿habéis oido hablar de la herencia?

(Que no estoy diciendo que lo debáis hacer así: ¿os repito lo de la entidad y objeto valor?)